pydicer/convert/data.py

```diff
@@ -70,6 +70,32 @@ def handle_missing_slice(files):
     else:
         raise ValueError("This function requires a Dataframe or list")
 
+    # If duplicated slices locations are present in series, check if the first duplicates have the
+    # same pixel data. If they do, then assume all are the same and drop the duplicates. Otherwise
+    # we raise an error.
+    df_duplicated = df_files[df_files["slice_location"].duplicated()]
+    if len(df_duplicated) > 0:
+        check_slice_location = df_duplicated.iloc[0].slice_location
```

Comment:
Does this check for *all* duplicate location or only checks for the first location we detect a duplication for?

Reply:
Thanks @dalmouiee, no it only checks the first. It assumes that if the first one had identical pixel values then they all will. Doing that for efficiency, and I think its quite safe? I can't think of an example where that wouldn't be the case, unless there was some corrupted data or something but that would be a different problem entirely..

Reply:
So what about the case that there happens to be other duplicates at different slice locations? Could that be a case?

Reply:
Yeah I mean anything is possible but I can't imagine a scenario where that would happen. It would be super rare, so I don't think the increased computation resources would be worth consuming every time here?

I suppose in a future version of pydicer, we could have different modes. So if someone super concerned about this kind of data integrity then they could run it in a "slow mode" or something where all this sort of stuff gets checked... But by default it would just behave like it is now...?

Reply:
Yep no worries sounds good!